lib/src/platform_client.dart

```diff
@@ -233,7 +235,6 @@ class PlatformClient {
       'requestType': 'AIRA', // Required but unused.
       'hasMessage': (null != message && message.isNotEmpty) || fileNames.isNotEmpty || true == cannotTalk,
       'message': '$message${fileNames.isEmpty ? '' : ' (With files: $fileNames)'}',
-      'fileIds': fileIds,
```

Comment:
I'm not sure why we were sending these to Platform, as it doesn't do anything with them.

Reply:
good catch! At first, I wanted to make Dash retrieve the URLs of the sent files within the dialog shown at call start. After having seen that this change in Dash wasn't as trivial, I decided to leave the filenames only on that dialog but forgot to remove the fileIds.